app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<% @contestants.each do |contestant| %>
+  <section id="contestant-<%= contestant.id %>">
+    <h1><%= contestant.name %></h1>
+
+    <p>Projects: <%= contestant.project_names %></p>
```

Comment:
This is supposed to be `<p>Projects: <%= contestant.project_names.joins(", ") %></p>` since the US has the project names in a comma-separated list.